src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -73,6 +85,11 @@
     private LinkedList<BlockPos> woodBlocks;
 
     /**
+     * All leaves of the tree.
+     */
+    private LinkedList<BlockPos> leaves;
```

Comment:
Yes I know, but since we want 1.10/1.11 possibly equal, if I change this we have a difference to 1.10 again